warp/yul/main.py

```diff
@@ -57,24 +57,35 @@ def transpile_from_solidity(sol_src_path, main_contract) -> dict:
     return {"cairo_code": cairo_code, "sol_abi": abi}
 
 
-def transpile_from_yul(yul_ast: ast.Node) -> str:
+def transpile_from_yul(yul_ast: ast.Node, optimizers_order="vcolrfd") -> str:
     name_gen = NameGenerator()
     cairo_functions = CairoFunctions(FunctionGenerator())
     builtins = get_default_builtins(cairo_functions)
+
     yul_ast = ForLoopSimplifier().map(yul_ast)
     yul_ast = ForLoopEliminator(name_gen).map(yul_ast)
     yul_ast = MangleNamesVisitor().map(yul_ast)
     yul_ast = SwitchToIfVisitor().map(yul_ast)
-    yul_ast = VariableInliner().map(yul_ast)
-    yul_ast = ConstantFolder().map(yul_ast)
-    yul_ast = FoldIf().map(yul_ast)
     yul_ast = ExpressionSplitter(name_gen).map(yul_ast)
```

Comment:
We want to run variable inliner atleast once before expresssion splitter, I think that will also fix your tests